apps/project/serializers.py

```diff
@@ -377,7 +381,7 @@ class ProjectStatSerializer(ProjectSerializer):
     number_of_leads_tagged = serializers.IntegerField(read_only=True)
     number_of_leads_tagged_and_verified = serializers.IntegerField(read_only=True)
     number_of_entries = serializers.IntegerField(read_only=True)
-    status = serializers.ReadOnlyField(source='status.title')
+    status = serializers.ReadOnlyField()
```

Comment:
We don't need this here. status is now a writable field.

Reply:
Done